ci/docker/python-wheel-musllinux.dockerfile

```diff
@@ -38,7 +38,7 @@ RUN apk add --no-cache \
     wget \
     zip
 # Add mono from testing repo because it's not in the main repo
```

Comment:
Could you update this comment too?

Note: We can merge this to the above `apk add` when we use Alpine Linux 3.22 or later.